src/chrome/content/rules/Python.org.xml

```diff
@@ -1,68 +1,76 @@
 <!--
-	Invalid certificate:
-		* es.python.org
-		* calendario.es.python.org
-		* documentos-asociacion.es.python.org
-		* empleo.es.python.org
-		* forum.pl.python.org
-		* hg.es.python.org (incomplete certificate chain)
-		* lists.es.python.org (incomplete certificate chain)
-		* mail.pl.python.org
-		* redesign.python.org
+	HSTS preloaded:
+		- docs
+		- doc
+		- hg
 
-	No working URL known:
-		* front.python.org
+	Incomplete certificate chain:
+		- hg.es
+		- lists.es
 
-	Refused:
-		* buildbot.python.org
```

Comment:
please keep this comment

Reply:
@J0WI Done.

Reply:
@J0WI Can you review as many PRs by me as possible? I won't be available for any corrections for 2 weeks.